### DISCUSSION TITLE
Docks at the bottom of Audacious do not save their sizes

### DISCUSSION THREAD

**Discussion by nixRidge** [audacious-media-player/audacious#1378] [General]
Hi, moved to this player a few days ago, loving it a lot!
Just checking in here to see if this is user error before I try to open an issue about it.
I have Album Art and Spectrum Analyser at the bottom of Audacious. But whenever I close Audacious, those positions and sizes aren't saved properly I think. The first screenshot is after I manually set the size the way I want, then in the second screenshot I've closed and re-opened Audacious (in a normal manner using the quit button. Using the cross in the window controls also causes this.)

Edit: This happens in Audacious 4.4-beta1, Qt 6.7, Wayland and Xwayland. I have three 1080p monitors (5760x1080 total resolution), and I use Plasma Wayland 6.1.

This is possibly relevant, but I'm not sure if this person is docking the Album Art view, or using it as a standalone window.
https://github.com/audacious-media-player/audacious/issues/1317

Anyone know about this issue, or have workarounds? Should I open a bug report about it? Thanks everyone!

![bilde](https://github.com/audacious-media-player/audacious/assets/40672277/f7ec3300-ab32-4319-a18f-88b57870e038)

![bilde](https://github.com/audacious-media-player/audacious/assets/40672277/f19ab2d9-3162-4272-baee-388f9464f491)

**Comment by radioactiveman** [comment]
Please tell us the version of Audacious and Qt.

I could not reproduce this with Audacious 4.4-beta1 and Qt 6.7. Can someone else reproduce the issue?

**Reply by nixRidge** [reply]
Right, bit embarrassing that I forgot to include that, lol.
This is Audacious 4.4-devel https://github.com/audacious-media-player/audacious/commit/494702f6a06137f432794612a99c94ae39107dd2 and plugins https://github.com/audacious-media-player/audacious-plugins/commit/78f33f7316de481d6ce34ed3614076fda700aa1c (built the day after those commits)
Edit: Rebuilt just now, just in case even tho there weren't any relevant changes, but same issue happens.

It also happens on both X11 and Wayland.
On Qt 6.7 as well.

**Comment by nixRidge** [comment]
I'm thinking it's some sort of issue with minimum size, see if I leave it at this size and restart Audacious, it stays like that.
![bilde](https://github.com/audacious-media-player/audacious/assets/40672277/6afb0c47-eb83-4293-a858-b9409a593187)

But if I make it really really thin and restart, it goes back to this size, **every** time
![bilde](https://github.com/audacious-media-player/audacious/assets/40672277/cfc1edfc-f9bc-4994-bb52-ff62387f1e82)
![bilde](https://github.com/audacious-media-player/audacious/assets/40672277/b7e91d06-886c-40b4-a191-a2b29bd28613)
